spec/cheetah_spec.rb

```diff
@@ -1,178 +1,172 @@
 require "spec_helper"
 require "fileutils"
 
-module Cheetah
-  describe DefaultRecorder do
-    before do
-      @logger = double
-      @recorder = DefaultRecorder.new(@logger)
+describe Cheetah::DefaultRecorder do
+  let(:logger) { double }
+  let(:recorder) { Cheetah::DefaultRecorder.new(logger) }
```

Comment:
np: Maybe you could use `subject(:recorder)` making it clear that it's the subject.